src/next/middleware/http-auth.ts

```diff
@@ -20,7 +45,9 @@ export function handleHttpBasicAuth (
 			.toString()
 			.split(":");
 
-		if (givenUser === username && givenPassword === password)
+		const hasValidUser = users.some((item : Credentials) => givenUser === item.username && givenPassword === item.password);
```

Comment:
```suggestion
		const hasValidUser = users.some(item => givenUser === item.username && givenPassword === item.password);
```

Don't need the type, as the array is already properly typed.